v1.1/information-schema.md

```diff
@@ -148,9 +148,9 @@ Column | Description
 `TABLE_TYPE` | Type of the table: `BASE TABLE` for a normal table, `VIEW` for a view, or `SYSTEM VIEW` for a view created by CockroachDB.
 `VERSION` | Version number of the table; versions begin at 1 and are incremented each time an `ALTER TABLE` statement is issued on the table.
 
-### user_privileges
+### user_privileges <div class="version-tag">New in v1.1</div>
```

Comment:
Sorry about that, I implied that you didn't want them to show up based off the screenshots in Asana. I can make it display there as well if that's what you'd like.